### PR TITLE
fixes/bugs-and-feedback

### PR DESCRIPTION
**Pull request by qsl-deepak**
## Title
Bugs and Feedback Fixes

## Description

- Covers fixes for bugs/feedbacks listed below:

 -ECER-2938
 -ECER-2721
 -ECER-3122
 -ECER-3123
 -ECER-3153
 -ECER-1794
 -ECER-3211
 -ECER-3177

## Related Jira Issue(s)

-ECER-2938
-ECER-2721
-ECER-3122
-ECER-3123
-ECER-3153
-ECER-1794
-ECER-3211
-ECER-3177

## Checklist
- [x] I have tested these changes locally.
- [ ] Changes to backend endpoints are covered by passing integration tests.
- [ ] I have added or updated the necessary documentation.

## Screenshots (if applicable)

ECER-2938:

image

ECER-2721:

localhost_5121_messages_ae52e6cb-487c-ef11-ac20-000d3a0a4c90_reply

ECER-3122:

localhost_5121

ECER-3123:

localhost_5121_requirements_5

ECER-3153:

localhost_5121_application_declaration

ECER-3211:

image

### CODE REVIEW

File: src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/components/pages/Application.vue [hunk 1]
```diff
@@ -26,6 +26,7 @@
               :title="step.title"
               :editable="index + 1 < wizardStore.step && wizardStore.listComponentMode !== 'add'"
               :complete="index + 1 < wizardStore.step"
+              style="white-space: nowrap; font-size: 0.9em"
```
Comment (pcatkins) [inline src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/components/pages/Application.vue:29]: Hey just for consistency can you please replace `nowrap` with:

`class="text-no-wrap"`

Which font size are you going for here? Can we use one of our classes from `typography.scss`?

**Review by pcatkins — CHANGES_REQUESTED** [review]
Small styling change request.

File: src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/components/pages/Application.vue (hunk 1 above)
Comment (qsl-deepak) [reply src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/components/pages/Application.vue]: Thanks for the review. addressed both concerns, using this now:
 :class="`small ${mdAndDown ? 'text-wrap' : 'text-no-wrap'}`"

**Review by pcatkins — DISMISSED** [review]

File: src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/utils/functions.ts [hunk 2]
```diff
@@ -152,3 +152,23 @@ export function sanitizeFilename(filename: string) {
...
+
+/**
+ * Scrolls to a given element using specified options for behavior, block, and inline alignment.
+ *
+ * @param {HTMLElement} element - The element to scroll to.
+ * @param {string} behavior - The scroll behavior ('auto', 'smooth').
+ * @param {string} block - The vertical alignment ('start', 'center', 'end', 'nearest').
+ * @param {string} inline - The horizontal alignment ('start', 'center', 'end', 'nearest').
+ */
+export function scrollToElement(element: HTMLElement, behavior: string = "auto", block: string = "start", inline: string = "nearest") {
```
Comment (SoLetsDev) [inline src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/utils/functions.ts:164]: Tiny comment we could type these as ScrollBehavior etc. 

Then you also get auto complete if you use the function? 

Could also default it to smooth if that's what we'll use most of the time.

Comment (qsl-deepak) [reply src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/utils/functions.ts]: Done!

**Review by SoLetsDev — APPROVED** [review]

**Pull request merged**